specs/dotsigstore_bundle/pb/sigstore_rekor.proto

```diff
@@ -0,0 +1,92 @@
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+package dev.sigstore.rekor.v1;
+
+option go_package = "github.com/sigstore/proto/rekor/v1";
+option java_package = "dev.sigstore.proto.rekor.v1";
+option java_multiple_files = true;
+option java_outer_classname = "RekorProto";
+
+// KindVersion contains the entry's kind and api version.
+message KindVersion {
+        // Kind is the type of entry being stored in the log.
+        // See here for a list: https://github.com/sigstore/rekor/tree/main/pkg/types
+        string kind = 3;
+        // The specific api version of the type.
+        string version = 4;
```

Comment:
why are these 3 & 4 and not 1 & 2?

Reply:
Good catch, this got unnoticed when I moved the kind/version into a separate object.